exp-player/addon/components/exp-exit-survey/component.js

```diff
@@ -121,23 +122,44 @@ export default ExpFrameBaseComponent.extend(Validations, FullScreen, {
             this.send('next');
         }
     },
-    currentSessionsCompleted: Ember.computed('frameContext', function() {
-        var pastSessions = this.get('frameContext.pastSessions');
-        if (pastSessions) {
-            return pastSessions.get('length') || 1;
```

Comment:
More explanation below, but here's how I would write the currentSessionStatus. It's more condensed and you don't have to loop through all your sessions twice in the beginning.  You only have to fetch the session's createdOn if that session had the min number of physics videos taken.  Not defining variables if they're only going to be used once. Uses some more modern ES6 JS formatting. Returns a dictionary instead of an array.

``` js
   currentSessionStatus: Ember.computed('frameContext.pastSessions', function() {
        let nSessions = 0;
        const sessionDates = [];
        this.get('frameContext.pastSessions').forEach(session => {
            let nVideos = 0;
            Object.keys(session.get('expData')).forEach(frameKeyName => {
                if (frameKeyName.includes('pref-phys-videos')) {
                    nVideos++;
                }
            });
            // Count only sessions with at least minVideos pref-phys-videos frames
            if (nVideos >= this.get('minVideosToCountSession')) {
                nSessions++;
                sessionDates.pushObject(moment(session.get('createdOn')));
            }
        });
        return {'nSessions': nSessions, 'daysSessionsCompleted': moment.max(sessionDates).diff(moment.min(sessionDates), 'days') + 1 };
    }),

```